tests/sqllogictests/suites/mode/cluster/distributed_delete.sql

```diff
@@ -0,0 +1,37 @@
+statement ok
+drop table if exists t;
+
+statement ok
+drop table if exists t_origin;
+
+# make sure there will be multiple blocks there, by shrink the `row_per_block`
+statement ok
+create table t (id int, c1 int, c2 int) row_per_block=10;
+
+# generate test data
+statement ok
+insert into t select number, number * 10, number * 5 from numbers(1000);
```

Comment:
:+1: 

how about splitting this insertion into three?  so that table of multiple segments could be convered.


~~~
insert into t select number, number * 10, number * 5 from numbers(500) where number ;
insert into t select number, number * 10, number * 5 from numbers(1000) where number > 499;
insert into t select number, number * 10, number * 5 from numbers(1500) where number > 1499;
~~~